app/error/httpErrors.js

```diff
@@ -21,7 +21,7 @@ export class ErrorWrongPassword extends HttpError {
 
 export class ErrorIncorrectUser extends HttpError {
   static throwHttp(req, res) {
-    res.status(404).json({ error: "User is incorrect." });
+    res.status(403).json({ error: "User is incorrect." });
```

Comment:
Not sure what this error was supposed to be initially. Perhaps ask Justin since he added it, but it might give insight as to why it was originally 404, in its original usage case.